js/components/utils.js

```diff
@@ -305,6 +305,53 @@ var Utils = {
     return isFinite(Number(property)) ? Number(property) : 0;
   },
 
+
+  /**
+   * Determines whether an element contains a class or not.
+   * TODO:
+   * classList.contains is much better for this purpose, but our current version
+   * of React Test Utils generates events with a null classList, which results in
+   * broken unit tests.
+   *
+   * @param {DOMElement} element The DOM element which we want to check
+   * @param {String} className The name of the class we want to match
+   * @return {Boolean} True if the element contains the given class, false otherwise
+   */
+  elementHasClass: function(element, className) {
+    if (!element) {
+      return false;
+    }
+    return (' ' + element.className + ' ').indexOf(' ' + className + ' ') > -1;
```

Comment:
This is in order to prevent a false positive when a class name contains the string that you're testing. For example, if you have an element with a class `fancyclass`, then doing `hasClass('fancy')` should return false